DiceKeys/Core/DiceKey/CanonicalizeJsonRecipe.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  CanonicalizeJsonRecipe.swift
+//  DiceKeys
+//
+//  Created by Angelos Veglektsis on 7/29/22.
+//
+
```

Comment:
To ensure those maintaining this code understand the interdependencies, please add:

```swift
// These functions should match the
// [Reference Implementation in TypeScript](https://github.com/dicekeys/dicekeys-app-typescript/blob/main/web/src/dicekeys/canonicalizeRecipeJson.ts)
// and its functionality should not be changed without ensuring that the reference implementation
// and dependent implementations are changed to match.
```

A similar comment should be placed into the equivalent Android code.

In the TypeScript code, please PR a comment
```TypeScript
// This is a reference implementation and should not be changed without changing
// the dependent implementations in
//   - [Swift](link here)
//   - [Kotlin](link here)
```

Please try to do that whenever you port TypeScript functionality so that we can better track where the inter-project dependencies.

Reply:
Fixed